src/main/java/io/github/cottonmc/cotton/gui/widget/WPlayerInvPanel.java

```diff
@@ -14,10 +14,10 @@
  * A player inventory widget that has a visually separate hotbar.
  */
 public class WPlayerInvPanel extends WPlainPanel {
-	private final WItemSlot inv;
-	private final WItemSlot hotbar;
+	protected final WItemSlot inv;
```

Comment:
1. Needlessly abbreviated names like `inv` shouldn't be exposed as APIs, it needs a human-readable name
2. Public APIs should have javadoc